### PR TITLE
Add move, flux and fork support to DSS Vat model

### PR DESCRIPTION
**Pull request by martinlsanchez**
This will add support for the following functions based on the current Vat contract implementation:

- `move`: transfer Dai between users
- `flux`: transfer collateral between users
- `fork`: split a Vault - binary approval or splitting/merging Vault's

Function descriptions based on [Maker Vat Glossary](https://github.com/makerdao/dss/wiki/Glossary).

### CODE REVIEW

**Comment by EdNoepel** [comment]
Much thanks.  Do you have capacity to update `tests/test_dss.py` to exercise these methods?

File: pymaker/dss.py [hunk 1]
```diff
@@ -384,6 +384,57 @@ def line(self) -> Rad:
...
+        """
+        assert isinstance(ilk, Ilk)
+        assert isinstance(src, Address)
+        assert isinstance(dst, Address)
+        assert isinstance(wad, Wad)
+
+        flux_args = [ilk.toBytes(), src.address, dst.address, wad.value]
+        return Transact(self, self.web3, self.abi, self.address, self._contract, 'flux', flux_args)
+
+    def move(self, src: Address, dst: Address, wad: Wad) -> Transact:
```
Comment (EdNoepel) [inline pymaker/dss.py]: The [contract](https://github.com/makerdao/dss/blob/master/src/vat.sol#L151) seems to take a `Rad`.

File: pymaker/dss.py [hunk 2]
```diff
@@ -384,6 +384,57 @@ def line(self) -> Rad:
...
+            wad: Amount of dai to move.
+        """
+        assert isinstance(src, Address)
+        assert isinstance(dst, Address)
+        assert isinstance(wad, Wad)
+
+        move_args = [src.address, dst.address, wad.value]
+        return Transact(self, self.web3, self.abi, self.address, self._contract, 'move', move_args)
+
+    def fork(self, ilk: Ilk, src: Address, dst: Address, dink: Wad, dart: Wad) -> Transact:
```
Comment (EdNoepel) [inline pymaker/dss.py]: How did you run the approvals required to `fork`?  I'd like to include those as well.

Comment (martinlsanchez) [reply pymaker/dss.py]: Are you talking about `hope` for the `wish` requirements?

File: pymaker/dss.py (hunk 1 above)
Comment (martinlsanchez) [reply pymaker/dss.py]: This is already fixed

**Pull request merged**